### ISSUE TITLE
Link without text needs ARIA label

### ISSUE THREAD

**Issue by emmalearnscode** [EddieHubCommunity/LinkFree#525]
### Description

The back arrow on personal profile pages doesn't have any text and therefore needs an aria label so that screen reader users know the purpose of the link and where it will take them

### Screenshots

<img width="538" alt="Screenshot 2021-10-17 201313" src="https://user-images.githubusercontent.com/57045550/137639754-7e6c6e95-31f0-426c-896f-d55749219127.png">

**Comment by JiaHua-Zou** [comment]
Hi, I would like to tackle this issue.
